config/initializers/redis.rb

```diff
@@ -0,0 +1,13 @@
+# Location: config/initializers/redis.rb
+def set_redis
+  conf_file = File.join('config', 'redis.yml')
+
+  if File.exists?(conf_file)
+    conf = YAML.load(File.read(conf_file))
+    conf[Rails.env.to_s].blank? ? Redis.new : Redis.new(conf[Rails.env.to_s])
+  else
+    Redis.new
+  end
+end
+
+$redis = set_redis
```

Comment:
Is this from the documentation, why do we need to do this? Usually calling `Redis.new` from w.e. class you're using it, is enough